### ISSUE TITLE
strange JS error initializing leveup

### ISSUE THREAD

**Issue by Ivshti** [trueinteractions/tint2#196]
This is the trace:

```
/Users/ivogeorgiev/linvo-cinematic/node_modules/stremio-local-files/node_modules/levelup/lib/errors.js:8
  , LevelUPError  = createError('LevelUPError')
                    ^

TypeError: createError is not a function
    at Object.<anonymous> (/Users/ivogeorgiev/linvo-cinematic/node_modules/stremio-local-files/node_modules/levelup/lib/errors.js:8:21)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/Users/ivogeorgiev/linvo-cinematic/node_modules/stremio-local-files/node_modules/levelup/lib/levelup.js:13:22)
```

The strange thing is that the same error doesn't appear when the module is required and used in the same way under node.js

``createError`` is traced to:
```
var createError   = require('errno').create
```

In tint2 requiring ``errno` is 
```
> require("errno")
[Function: errno]
> require("errno").create
undefined
```

While in Node.js it's a long object with the ``.create`` set

--------

How to check if ``errno` is required from a different paths in both cases?

----

node v4.4.3
tint2 v5.9.0

**Comment by trevorlinton** [comment]
It appears the wrong module is being loaded, errno is a built in module for FFI error handling in Tint, we can rename it as its only a downstream provider module, this will be with the next release 5/30. 

See: https://github.com/trueinteractions/tint2/blob/master/modules/Bridge/errno.js

**Comment by Ivshti** [comment]
OK, so I've updated the title

Also, any way to work around this? Requiring the module by its direct path?

Shouldn't it be better if all tint2-specific modules were prefixed?

**Comment by trevorlinton** [comment]
This is fixed in v2.3.9, let me know if you continue to have issues.
